model/mmwave-sidelink-spectrum-phy.cc

```diff
@@ -271,16 +282,20 @@ MmWaveSidelinkSpectrumPhy::StartRxData (Ptr<MmWaveSidelinkSpectrumSignalParamete
       // If there are other intereferent devices that transmit in the same slot, the current
       // device simply does not consider the signal and goes on with the transmission. The code does not raise any errors since
       // otherwise we are not able to study scenarios where interference could be an issue.
-      break;
+      //break;
     case RX_CTRL:
-      NS_FATAL_ERROR ("Cannot receive control in data period");
-      break;
+      //NS_FATAL_ERROR ("Cannot receive control in data period");
```

Comment:
Anche se non viene utilizzato, è utile lasciare l'istruzione di errore. In questo modo, se per qualsiasi ragione entri in uno stato sbagliato, puoi risalire all'errore facilmente. 
Altrimenti, potresti perderci un bel po' di tempo.